### PR TITLE
Solved Incorrect Output of More Command On a Short File

### PR DESCRIPTION
**Pull request by MrSquanchee**
Removed endwin() call before the display of file contents.
Modified code to properly detect EOF of the given file.

### CODE REVIEW

File: src/cmds/fs/more.c [hunk 1]
```diff
@@ -28,29 +28,31 @@ static void screen(FILE *fp) {
...
+	//endwin();
 
 	while (1) {
 		for (x = 0; x < lines - 1; x++) {
 
 			for (y = 0; y < columns; y++) {
-				buff[y] = getc(fp);
-				switch ((int) buff[y]) {
-				case EOF:
+				if(fread(&buff[y],1,1,fp)==1){
```
Comment (Deryugin) [inline src/cmds/fs/more.c]: Please follow our [code style](https://github.com/embox/embox/wiki/Code-Style), you're missing some spaces here.

File: src/cmds/fs/more.c [hunk 2]
```diff
@@ -28,29 +28,31 @@ static void screen(FILE *fp) {
...
+							y = columns;
+							break;
+						case '\t':
+							/*	Perform tab instert	*/
+							memset(buff + y, ' ', TAB_SIZE);
+							y += TAB_SIZE - 1;
+							break;
+					}
+				}
+				else{
```
Comment (Deryugin) [inline src/cmds/fs/more.c]: Code style issue: `else {` should be placed on the same line with previous `}`

File: src/compat/posix/curses/curses.c [hunk 3]
```diff
@@ -109,7 +109,7 @@ static SCREEN * newterm(char *type, FILE *outfile, FILE *infile) {
 	screen.out = outfile;
 	screen.in = infile;
 	window_init(&screen.std_win, 0, 0, LINES, COLS, NULL, &screen.std_buff[0][0]);
-	memset(&screen.std_buff[0][0], ' ', sizeof screen.std_buff);
+	memset(&screen.std_buff[0][0],0, sizeof screen.std_buff);
```
Comment (Deryugin) [inline src/compat/posix/curses/curses.c]: Missing space between ',' and '0'

**Comment by MrSquanchee** [comment]
I will try this with the appropriate code style

**Pull request closed without merging**